modules/jolt_physics/config.py

```diff
@@ -1,5 +1,11 @@
 def can_build(env, platform):
-    return not env["disable_3d"]
+    if env["disable_3d"]:
+        return False
+
+    if env["arch"] in ["loongarch64", "ppc32", "ppc64", "rv64"]:
+        return False
+
+    return True
```

Comment:
Alternatively, we could enable it on supported architectures.

```suggestion
    return env["arch"] in ["x86_32", "x86_64", "arm32", "arm64", "wasm32"]:
```